CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 #### Fixes
 
 * Your contribution here.
+* [#71](https://github.com/ruby-grape/grape-swagger-entity/pull/71): Fix regression bug for enum values in array attributes - [@Jell](https://github.com/Jell)
```

Comment:
Let's get rid of the word "bug", since a regression _is_ a bug :) 

"Fix: regression for enum values in array attributes"

The line needs a period so danger doesn't complain.

Reply:
ah! right 😅 fixed now!